jaeger_client/sampler.py

```diff
@@ -264,8 +264,8 @@ def __init__(self, strategies, max_operations):
         self.max_operations = max_operations
 
     def is_sampled(self, trace_id, operation=''):
-        sampler = self.samplers.get(operation, None)
```

Comment:
I thought the reason we pass the default value is because the dictionary might otherwise blow up if the key is not present. Or is it only if we use brackets `sampler[operation]`?

In any case, I don't think that the get() statement itself needs to change, according to your explanation it was more that the `if X is not None` checks that were tripping the error.

Reply:
Actually Python defaults to `None` if no second argument is passed so this is just unnecessary (see https://docs.python.org/2/library/stdtypes.html#dict.get). I fixed all places in these files where I found that. If you want me to revert the unrelated changes I can.

Reply:
nah, it's fine

Reply:
:)